.github/workflows/test.yml

```diff
@@ -1,19 +0,0 @@
-name: test
-on: [push, pull_request]
-
-jobs:
-  test:
-    name: Node.js ${{ matrix.node_version }}
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node_version: [10, 12]
-
-    steps:
-      - uses: actions/checkout@master
-      - name: Use Node.js ${{ matrix.node_version }}
-        uses: actions/setup-node@v1
-        with:
-          node-version: ${{ matrix.node_version }}
-      - run: npm install
-      - run: npm test
```

Comment:
Was `npm test` deleted altogether? (From workflows, I know it's still in `package.json`.)

Reply:
I understand now. The `test` command is in the generated app. The generator only has a linter and it was a misnomer here.

Reply:
You can actually see the workflow runs on my fork [here](https://github.com/shahoob/create-ink-app/actions).